<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nostoSummaWindow</class>
 <widget class="QWidget" name="nostoSummaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1579</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nosto</string>
  </property>
  <widget class="QPushButton" name="pushButton20e">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>440</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton40e">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>440</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton60e">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>530</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton100e">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>530</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton200e">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>620</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton500e">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>620</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>320</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Valitse nostettava summa:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suljeButton">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>770</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
  </widget>
  <widget class="QLabel" name="nosto_info">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kyhny_info">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>210</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nimi_label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>200</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>nimi_label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kuittiButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>710</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kuitti</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muuButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>440</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Muu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>520</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="muu_info">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>370</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Syötä summa ja paina OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="jakolabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Nostoja voi tehdä vain 10 euron välein!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>730</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timer">
   <property name="geometry">
    <rect>
     <x>1473</x>
     <y>872</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nollaButton">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>730</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yksiButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>580</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kaksiButton">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>580</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kolmeButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>580</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="neljaButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>630</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viisiButton">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>630</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kuusiButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>630</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seitsemanButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>680</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kahdeksanButton">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>680</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yhdeksanButton">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>680</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kumiButton">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>730</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
